decentdao/decent-interface
# `[ENG-65]` `[ENG-66]` Create example templates for Transfer,  Airdrop and Sablier stream creation

## Changes

This PR contains following functional changes
- Implement "Example Templates" and display it for every DAO on any chain. Currently - there're 3 templates: "Transfer", "Airdrop Tokens", "Create Sablier Stream"
- Implement "Airdrop Tokens" feature - it mostly replicates "Send Assets" experience. Filling and submitting the form in the modal will take user to the "Proposal With Actions" form with the ability to add more actions to the proposal and fill in title, description and documentation link
- Example template for "Transfer" simply leverages existing "Send Assets" UI / UX and replicates behavior for "Request Payment" on "Treasury" page
- Example template for "Create Sablier Stream" simply navigates to "Create Sablier Proposal" UI.


## Testing

1. Navigate to DAO's Proposal Templates page (from the Dashboard: "Create Proposal" -> "Browse Templates")
2. Verify that new section with "Pre-configured Templates" available for any DAO on any chain. It should have 3 templates: "Transfer", "Airdrop Tokens", "Create Sablier Stream"
3. Verify that "Transfer" template correctly opens "Send Assets Modal", filling and submitting form should navigate to "Create Proposal With Actions" form, where "Send Assets" action will be added in the actions list
4. Verify that "Airdrop Tokens" template correctly opens respective modal, filling and submitting form should navigate to "Create Proposal With Actions" form, where "Airdrop" action will be added in the actions list
5. Verify that "Create Sablier Stream" navigates to "Create Sablier Proposal". Filling the form immediately submits the proposal with filled title, description, documentation link and respective streams. For now it supports only tranched streams creation

## Issues
- Closes #2533 
- Closes #2456 

## Screenshots

<img width="1728" alt="Screenshot 2024-12-20 at 23 41 32" src="https://github.com/user-attachments/assets/cd70791d-956d-433d-8783-f7fc93e44d7e" />
